### PR TITLE
Privatechain fixtures simplification

### PR DESCRIPTION
**Pull request by hackaugusto**
merge after #1666

### CODE REVIEW

File: raiden/tests/utils/network.py [hunk 1]
```diff
@@ -298,3 +320,159 @@ def create_apps(
...
+        poll_timeout,
+        web3=None,
+):
+    deploy_blockchain = BlockChainService(
+        deploy_key,
+        deploy_client,
+        GAS_PRICE,
+    )
+
+    secret_registry_address = deploy_contract_web3(
```
Comment (hackaugusto) [inline raiden/tests/utils/network.py]: next step is to move the deployment of these smart contracts to separate functions and fixtures

File: raiden/tests/conftest.py [hunk 2]
```diff
@@ -68,40 +67,6 @@ def pytest_addoption(parser):
...
-
-def load_fixtures_list(module_list: str) -> Dict:
-    fixtures_all = {}
-    for module in module_list:
-        fixtures_all.update(load_fixtures(module))
-    return fixtures_all
-
-
-@pytest.hookspec()
-def pytest_configure(config):
```
Comment (hackaugusto) [inline raiden/tests/conftest.py]: this required global state changes + fixtures of a separate repo, I removed this to streamline the usage of the fixtures (they already have many hops, and having to search for fixtures on multiple repos is not the best approach)

File: raiden/tests/utils/geth.py [hunk 3]
```diff
@@ -129,34 +140,36 @@ def geth_create_account(datadir, privkey):
...
+def geth_generate_poa_genesis(
+        genesis_path,
+        accounts_addresses,
+        seal_address,
+        random_marker,
+):
     """Writes a bare genesis to `genesis_path`.
 
     Args:
         genesis_path (str): the path in which the genesis block is written.
```
Comment (palango) [inline raiden/tests/utils/geth.py]: Would be nice to move the type annotations out of the docstring into the function signature.

File: raiden/tests/utils/geth.py [hunk 4]
```diff
@@ -339,8 +339,74 @@ def geth_create_blockchain(
...
+        accounts_to_fund,
+        geth_nodes,
+        base_datadir,
+        verbosity,
+        random_marker,
+):
+    """ Starts a private network with private_keys accounts funded.
+
+    Args:
+        web3 (Web): A Web3 instance used to check when the private chain is running.
```
Comment (palango) [inline raiden/tests/utils/geth.py]: Type annotations again

File: raiden/tests/utils/geth.py [hunk 5]
```diff
@@ -90,7 +101,7 @@ def geth_to_cmd(node, datadir, verbosity):
         '--rpcapi', 'eth,net,web3',
         '--rpcaddr', '0.0.0.0',
         '--networkid', '627',
-        '--verbosity', str(verbosity),
+        '--verbosity', '3',
```
Comment (palango) [inline raiden/tests/utils/geth.py]: Why this change? If it's not needed pls remove the `verbosity`  parameter from the function

File: raiden/tests/integration/api/test_pythonapi.py [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 from raiden.api.python import RaidenAPI
 from raiden.tests.utils.transfer import get_channelstate
-from raiden.tests.utils.blockchain import wait_until_block
+from raiden.tests.utils.geth import wait_until_block
```
Comment (palango) [inline raiden/tests/integration/api/test_pythonapi.py]: I feel we should leave `wait_until_block` in `raiden.tests.utils.blockchain`, it's not related to geth.

Comment (palango) [reply raiden/tests/integration/api/test_pythonapi.py]: After discussion with Augusto we postponed this into another PR.
